.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
we might also add/enable isort and add-tailing-comma to the pre-commit CI
ref : https://github.com/ansible-collections/cisco.ios/blob/main/.pre-commit-config.yaml